src/openzeppelin/access/ownable/library.cairo

```diff
@@ -1,111 +1,90 @@
-# SPDX-License-Identifier: MIT
-# OpenZeppelin Contracts for Cairo v0.3.2 (access/ownable/library.cairo)
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.3.2 (access/ownable/library.cairo)
 
 %lang starknet
 
 from starkware.starknet.common.syscalls import get_caller_address
 from starkware.cairo.common.cairo_builtins import HashBuiltin
 from starkware.cairo.common.math import assert_not_zero
 
-#
-# Events
-#
+//
+// Events
+//
 
 @event
-func OwnershipTransferred(previousOwner: felt, newOwner: felt):
-end
+func OwnershipTransferred(previousOwner: felt, newOwner: felt) {
+}
```

Comment:
what do you think about inlining these?

Reply:
v0.10.0 uses an opinionated formatter that everyone will need to use : 

no more different styles and flavours imo

Reply:
@ExoMonk why is it mandatory?

Reply:
Starknet Compiler doesn't allow you multiple bracket styles and therefore, as of 0.10.0, bracket styles are fixed : 
let's imagine we want to define the simplest storage var or event, we must ensure ending with a newline closing bracket such as : 

```
@storage_var
func my_var() -> (_var: felt) {
}
```

Reply:
very weird. is there any alleged reason for this?

Reply:
@martriay I would say for that logic is now much easuer to parse => performance 

Which is what we need for SC

Reply:
i don’t think this should impact performance in any level. this is trivial computation for a parser (every language does it) and should have no impact on runtime execution (it’s not an interpreted language). are you sure it's performance?

Reply:
Guessing, but for sure it improves logic parsing

Starkware team said : brackets replace now « end » blocks, and the way Cairo is build is case sensitive, therefore end of brackends is « literaly » replacing « end »